src/utils/helpers/browser/browser.js

```diff
@@ -60,6 +60,16 @@ const Browser = {
   },
 
   /**
+   * Focues and sets cursor of input filed but does not select text
+   *
+   * @method setFocus
+   */
+  setFocus: (ref) => {
+    let node = ReactDOM.findDOMNode(ref._input);
```

Comment:
Made the change suggested by @SeanArmstrong.  Also added the safe guard for linting error in both `editFocus` and `setFocus` methods.